src/components/modules/Ido/IdoWarning.tsx

```diff
@@ -1,6 +1,5 @@
-import { WarningIcon } from '@chakra-ui/icons'
+import { Icon } from '@chakra-ui/icons'
 import { Flex, Image } from '@chakra-ui/react'
-import { useState } from 'react'
 
 interface IdoWarningProps {
```

Comment:
The warning module can use across all pages on the website. So please put it into a common module.

Reply:
fixed